_FIPS201/requirements.md

```diff
@@ -406,13 +406,15 @@ If the expiration date of the new PIV Card is later than the expiration date of
 about the cardholder is being changed, the card issuer **SHALL** ensure that an adjudicative entity has authorized
 the issuance of the new PIV Card. The issuer **SHALL** ensure that the adjudicative entity has verified that there is a PIV eligibility determination in an authoritative record, such as the agency's IDMS or the Central Verification System (or successor).
 
-The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card or PIV enrollment record. Minimum accuracy requirements for
+The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card[^OCCorBIO] or PIV enrollment record. Minimum accuracy requirements for
 the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On a positive biometric verification decision, the new PIV Card **SHALL** be
 released to the applicant. If the biometric verification decision is negative, or if no biometric data records are available, the cardholder
 **SHALL** provide two identity source documents (as specified in [Section 2.7](requirements.md#s-2-7)), and an attending operator **SHALL**
 inspect these and compare the cardholder with the electronic facial image retrieved from the enrollment data record
 and the photograph printed on the new PIV Card.
 
+[^OCCorBIO]: BIO-A or OCC-AUTH authentication mechanism is performed when using the PIV Card for biometric  verification. 
```

Comment:
But not BIO then, right? That makes this a normative requirement, and should be therefore inline in text.